Library/Formula/qt5.rb

```diff
@@ -72,6 +74,13 @@ class Qt5 < Formula
   depends_on OracleHomeVarRequirement if build.with? "oci"
 
   def install
+    # Save users some frustration by telling them that `--cc=gcc-5` or similar
+    # won't work. It won't be picked up by the Qt build system, there is no easy
+    # way to select an arbitrary compiler, and it's not supported anyway.
+    if ENV.compiler != :clang
```

Comment:
> Is this a bug or by design?

It's perhaps by design but I'd happily see that "design" changed.



> I'm not sure if this is a feasible project. A lot of compiler details seem to be baked in into those mkspecs and I can easily imagine things to break if superenv is forced on the Qt build system.

I'd be surprised if this breaks that badly. I suspect it's a full vs relative path thing.